RecoLocalTracker/SiPixelRecHits/python/SiPixelRecHits_cfi.py

```diff
@@ -29,6 +29,8 @@
 siPixelRecHitsPreSplittingTask = cms.Task(siPixelRecHitsPreSplitting)
 
 siPixelRecHitsCUDAPreSplitting = _siPixelRecHitCUDA.clone()
+siPixelRecHitsCUDAPreSplitting.beamSpot = cms.InputTag("offlineBeamSpotToCUDA")
```

Comment:
I'd suggest either
```suggestion
siPixelRecHitsCUDAPreSplitting = _siPixelRecHitCUDA.clone(
    beamSpot = "offlineBeamSpotToCUDA"
)
```
or change the default in
https://github.com/cms-patatrack/cmssw/blob/0c377ca56a12e7331c7434301e1c5949992983ee/RecoLocalTracker/SiPixelRecHits/plugins/SiPixelRecHitCUDA.cc#L59